### ISSUE TITLE
No space between Nerd font icon and label 

### ISSUE THREAD

**Issue by etienne-monier** [polybar/polybar#2243]
## The issue
There is no space between the Nered font icon and the text.

![bug_polybar](https://user-images.githubusercontent.com/32652388/99259913-ff27ef00-281a-11eb-8bc4-27763b7804dc.png)

**Expected behavior:**

My config worked well till now. I've defined a label containing an icon + a space + a stat (e.g. cpu usage), cf the above picture or [here](https://github.com/etienne-monier/dotfiles/blob/master/config/polybar/.config/polybar/config). Normally, there should be a space between them.

**Actual behavior:**

It seems there is a negative space. If I add some spaces in the label description, it looks good, but that's strange. 

**Was it working before?**

It worked perfectly in a previous 16.04 and 18.04 LTS computer. I installed it on my new 20.04 LTS and the bug appeared, but around three weeks after... 

## To Reproduce

My config is  [here](https://github.com/etienne-monier/dotfiles/blob/master/config/polybar/.config/polybar/config).

I'm using polybar 3.4.0-120-g3895ace.

## Polybar Log

The log at trace level ...

```
notice: Parsing config file: /home/etienne/.config/polybar/config
- config_parser: Parsing /home/etienne/.config/polybar/config
- config: Copying missing params (sub="bar/bottom", base="bar/default")
- config: Copying missing params (sub="bar/homelaptop", base="bar/bottom")
- config: Copying missing params (sub="bar/left", base="bar/default")
- config: Copying missing params (sub="bar/mainlaptop", base="bar/bottom")
- config: Copying missing params (sub="bar/right", base="bar/default")
- config: Copying missing params (sub="bar/smalllaptop", base="bar/bottom")
- config: Copying missing params (sub="module/backlight-acpi", base="module/xbacklight")
* Loaded monitor HDMI-1 (1920x1080+0+0)
* Bar geometry: 1920x30+0+1050; Borders: 0,0,0,0
- bar: Attach X event sink
- bar: Attach signal receiver
- controller: Install signal handler
- controller: Setup user-defined modules
- module/network: Deconstructing
error: Disabling module "network" (reason: Invalid network interface "enp0s25")
- pulseaudio: started mainloop
notice: pulseaudio: using default sink alsa_output.pci-0000_28_00.4.analog-stereo
- module/battery: Deconstructing
error: Disabling module "battery" (reason: No suitable way to get current charge state)
- module/powermenu: Creating menu level 0
- module/powermenu: Creating menu level item 0
- module/powermenu: Creating menu level item 1
- module/powermenu: Creating menu level item 2
* Starting application
- controller: Main thread id = 1
* Starting module/i3
* Starting module/date
- module/i3: Thread id = 2
* Starting module/cpu
* Starting module/memory
- module/cpu: Thread id = 3
* Starting module/temperature
- module/memory: Thread id = 4
* Starting module/pulseaudio
- module/temperature: Thread id = 5
* Starting module/powermenu
- module/powermenu: Thread id = 6
* Entering event loop (thread-id=1)
- module/pulseaudio: Thread id = 8
* Eventqueue worker (thread-id=7)
- bar: Create renderer
- renderer: Get TrueColor visual
- renderer: Allocate colormap
- renderer: Allocate output window
- renderer: Allocate window pixmaps
- renderer: Allocate graphic contexts
- renderer: Allocate alignment blocks
- renderer: Allocate cairo components
- renderer: Load fonts
* Configured DPI = 96x96
* module/date: Invoking shell command: "/home/etienne/.config/polybar/scripts/date/date.sh"
- command: Waiting for pid 12322 to finish...
- command: Exited with status 0
notice: Loaded font "System San Francisco Display:size=10" (name=DejaVu Sans, offset=2, file=/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf)
notice: Loaded font "Hack Nerd Font:style=Regular:size=11" (name=Hack Nerd Font, offset=2, file=/home/etienne/.local/share/fonts/NerdFonts/Hack Regular Nerd Font Complete.ttf)
* Bar window: 0x1600002
- bar: Reconfigure window
- bar: Set window WM_NAME
- bar: Set window _NET_WM_WINDOW_TYPE
- bar: Set window _NET_WM_STATE
- bar: Set window _NET_WM_DESKTOP
- bar: Set window _NET_WM_PID
- bar: Map window
- bar: Draw empty bar
- bar: Setup tray manager
- tray: Find systray selection atom for the default screen
* Activating tray manager
- tray: Create tray window
* Tray window: 0x1600006
- tray: Restacking tray window
- bar: Set window WM_NAME / WM_CLASS
- tray: Set window WM_PROTOCOLS
- tray: Set window _NET_WM_WINDOW_TYPE
- tray: Set window _NET_WM_STATE
- tray: Set window _NET_WM_PID
- tray: Set window _NET_SYSTEM_TRAY_VISUAL
- tray: Set window _NET_SYSTEM_TRAY_ORIENTATION
- tray: Set _NET_SYSTEM_TRAY_COLORS to ff222222
warn: Systray selection already managed (window=0x1400006)
- tray: Listen for events on the new selection window
* Deactivating tray manager
- tray: Unembed clients
- tray: Destroy window
* module/i3: Rebuilding cache
* module/date: Rebuilding cache
* module/cpu: Rebuilding cache
* module/memory: Rebuilding cache
* module/temperature: Rebuilding cache
* module/pulseaudio: Rebuilding cache
* module/powermenu: Rebuilding cache
- bar: Force update
* Redrawing bar window
- renderer: flush(1 geom=56x30+0+0, falloff=0)
- renderer: flush(2 geom=228x30+846+0, falloff=0)
- renderer: flush(3 geom=417x30+1503+0, falloff=0)
- bar: Force update
* Redrawing bar window
- renderer: flush(1 geom=56x30+0+0, falloff=0)
- renderer: flush(2 geom=228x30+846+0, falloff=0)
- renderer: flush(3 geom=417x30+1503+0, falloff=0)
- bar: Received expose event
* module/temperature: Rebuilding cache
* Redrawing bar window
- renderer: flush(1 geom=56x30+0+0, falloff=0)
- renderer: flush(2 geom=228x30+846+0, falloff=0)
- renderer: flush(3 geom=417x30+1503+0, falloff=0)
* module/cpu: Rebuilding cache
* module/memory: Rebuilding cache
* module/temperature: Rebuilding cache
* Redrawing bar window
- renderer: flush(1 geom=56x30+0+0, falloff=0)
- renderer: flush(2 geom=228x30+846+0, falloff=0)
- renderer: flush(3 geom=417x30+1503+0, falloff=0)
* module/temperature: Rebuilding cache
* Redrawing bar window
- renderer: flush(1 geom=56x30+0+0, falloff=0)
- renderer: flush(2 geom=228x30+846+0, falloff=0)
- renderer: flush(3 geom=417x30+1503+0, falloff=0)
* module/cpu: Rebuilding cache
* module/memory: Rebuilding cache
* module/temperature: Rebuilding cache
* Redrawing bar window
- renderer: flush(1 geom=56x30+0+0, falloff=0)
- renderer: flush(2 geom=228x30+846+0, falloff=0)
- renderer: flush(3 geom=417x30+1503+0, falloff=0)
notice: Termination signal received, shutting down...
- controller: Uninstall sighandler
- controller: Detach signal receiver
- controller: Stop modules
* module/i3: Disconnecting from socket
* module/i3: Stopping
warn: module/i3: Attempting to reconnect socket (reason: Unexpected EOF while reading header)
* module/i3: Reconnecting socket succeeded
- module/i3: Release sleep lock
* Deconstruction of module/i3 took 0 ms.
- module/date: Release sleep lock
* module/date: Stopping
- module/date: Release sleep lock
* Deconstruction of module/date took 0 ms.
* module/cpu: Stopping
- module/cpu: Release sleep lock
* Deconstruction of module/cpu took 0 ms.
* module/memory: Stopping
- module/memory: Release sleep lock
* Deconstruction of module/memory took 0 ms.
* module/temperature: Stopping
- module/temperature: Release sleep lock
* Deconstruction of module/temperature took 0 ms.
* module/pulseaudio: Stopping
- module/pulseaudio: Release sleep lock
* Deconstruction of module/pulseaudio took 0 ms.
* module/powermenu: Stopping
- module/powermenu: Release sleep lock
* Deconstruction of module/powermenu took 0 ms.
- controller: Joining threads
- module/i3: Deconstructing
- module/date: Deconstructing
- module/cpu: Deconstructing
- module/memory: Deconstructing
- module/temperature: Deconstructing
- module/pulseaudio: Deconstructing
- module/powermenu: Deconstructing
* Waiting for spawned processes to end
* Reached end of application...

```

## Environment:

* WM: i3wm
* Distro: Ubuntu 20.04
* Output of `polybar -vvv`:

```
polybar 3.4.0-120-g3895ace

Features: +alsa +curl +i3 +mpd +network(wireless-tools) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm -xcursor

Build type: Release
Compiler: /usr/bin/g++
Compiler flags:  -Wall -Wextra -Wpedantic -O3 -DNDEBUG
Linker flags:  
```

**Comment by patrick96** [comment]
I think this is the same issue as reported in #991. I have added some of my findings to the original post. 

TL;DR: Likely Nerd Fonts issue and not polybar.

**Comment by etienne-monier** [comment]
Ok ! As many, I just moved Nerd font back to v2.0.0. That worked great !
